gulpfile.js

```diff
@@ -1,24 +1,37 @@
 #!/usr/bin/env node
 
 'use strict';
-var { task } = require('gulp');
-var fs = require('fs');
+const { task } = require('gulp');
+const fs = require('fs');
 const util = require('util');
 const promisifyExec = util.promisify(require('child_process').exec);
 
 var packageJSON = require('./package.json');
 
+/**
+ * @dev Task to create git tag using version from package.json and pushing this specific tag
+ */
 task('publishTag', async function () {
-    await promisifyExec("git tag v"+ packageJSON.version +"; git push --tags");
+    const tag = "v" + packageJSON.version
+    await promisifyExec("git tag "+ tag +"; git push origin "+ tag);
 });
 
+/**
+ * @dev Task to update changelog for latest release
+ */
 task('updateChangelog', async function () {
-    let previous_version = process.argv[4];
-    let next_version = "v" + packageJSON.version;
+    const previous_version = process.argv[4];
+    const next_version = "v" + packageJSON.version;
+
+    // Create changes.md with latest release changelog temporarily
     await promisifyExec("github-changes -o ethereum -r remix -a --file changes.md --only-pulls --use-commit-body --only-merges --between-tags " + previous_version + "..." + next_version);
-    let data = fs.readFileSync(__dirname + '/changes.md', 'utf8') + '\n\n' + fs.readFileSync(__dirname + '/CHANGELOG.md', 'utf8')
+    // Concatenate new changelog content to the top of old changelog file content
+    const data = fs.readFileSync(__dirname + '/changes.md', 'utf8') + '\n\n' + fs.readFileSync(__dirname + '/CHANGELOG.md', 'utf8')
```

Comment:
Same as above. Also maybe a good idea to create intermediary variables for 
- fs.readFileSync(__dirname + '/changes.md', 'utf8')
- fs.readFileSync(__dirname + '/CHANGELOG.md', 'utf8')